Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsBuildProvider.cs

```diff
@@ -9,21 +9,31 @@ namespace UnityEngine.InputSystem.Editor
 {
     internal class InputSettingsBuildProvider : IPreprocessBuildWithReport, IPostprocessBuildWithReport
     {
+        InputActionAsset m_ProjectWideActions;
+        Object[] m_OriginalPreloadedAssets;
         public int callbackOrder => 0;
 
         public void OnPreprocessBuild(BuildReport report)
         {
-            if (InputSystem.settings == null)
-                return;
+            m_OriginalPreloadedAssets = PlayerSettings.GetPreloadedAssets();
+            var preloadedAssets = PlayerSettings.GetPreloadedAssets();
```

Comment:
Could do this to avoid calling playersettings function twice as it may have higher cost.

 var preloadedAssets = m_OriginalPreloadedAssets;